lib/jobs/ipmi-catalog.js

```diff
@@ -110,11 +110,26 @@ function ipmiCatalogJobFactory(
         assert.string(command);
         assert.object(obmSettings);
 
-        return [ 
+        // Some commands end with " || true"
+        // This tail needs to be deleted from the command args
+        // Otherwise the tail passed to ipmitool child process will cause error response
+        // this tail is stored and used to determine whether continue sending other commands
+        var ignore = false;
+        var tailIdx = command.indexOf(' ||');
```

Comment:
Why not only change the command to remove the "|| true" in the task definition of [catalog-mgmt-bmc.js](https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/catalog-mgmt-bmc.js). I think this kind of "smart' conversion is not necessary and waste.